### PR TITLE
edits to digital update script

### PR DESCRIPTION
**Pull request by pc37utn**

### CODE REVIEW

File: digital-islandupdate.sh [hunk 1]
```diff
@@ -104,6 +104,9 @@ mv ./pdf.js ./pdfjs
 rm -R jstree
 git clone https://github.com/vakata/jstree.git
 
+# JAIL library
+rm -R JAIL
```
Comment (markpbaggett) [inline digital-islandupdate.sh:108]: @pc37utn, ignore this comment.  for some reason was thinking you wanted to drop JAIL, but then remembered that was JW Player.

File: digital-islandupdate.sh [hunk 2]
```diff
@@ -123,7 +126,7 @@ git clone https://github.com/utkdigitalinitiatives/islandora_datastream_replace
 rm -R islandora
 git clone git://github.com/Islandora/islandora
 
-#- islandora_solution_pack_collection
+#- our version of islandora_solution_pack_collection
```
Comment (markpbaggett) [inline digital-islandupdate.sh:129]: Just wondering, why do we keep our own copy of this?  I know we modify the .tpls, but those are overwritten in the theme layer, not in the actual module.  Just trying to remember.

**Review by markpbaggett — COMMENTED** [review]
Everything else looks good to me.  Just wondering if we know why we keep our own basic collection module.

File: digital-islandupdate.sh (hunk 2 above)
Comment (pc37utn) [reply digital-islandupdate.sh:129]: the default sort of listings is by title,  we replace that by a sort on PID

Comment (markpbaggett) [reply digital-islandupdate.sh:129]: yeah, just found it: https://github.com/utkdigitalinitiatives/islandora_solution_pack_collection/commit/081523e4405eb6b0a9ea6bac300adf8742a4145d

Comment (markpbaggett) [reply digital-islandupdate.sh:129]: So i'm good. Merging.

**Pull request merged**